package.json

```diff
@@ -2,19 +2,30 @@
   "name": "multihashes",
   "version": "0.4.13",
   "description": "multihash implementation",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "lint": "aegir lint",
+    "lint": "lint-staged && npm run type-check",
     "test:browser": "aegir test --target browser",
     "test:node": "aegir test --target node",
-    "build": "aegir build",
-    "test": "aegir test",
+    "type-check": "flow check",
+    "build": "npm run build:node && BABEL_ENV=umd aegir build",
+    "test": "npm run build:node && aegir test",
     "docs": "aegir docs",
     "release": "aegir release --docs",
     "release-minor": "aegir release --type minor --docs",
     "release-major": "aegir release --type major --docs",
     "coverage": "aegir coverage",
-    "coverage-publish": "aegir coverage --provider coveralls"
+    "coverage-publish": "aegir coverage --provider coveralls",
+    "build:types": "flow-copy-source --verbose src lib",
+    "build:lib": "babel --out-dir lib src",
+    "build:node": "npm run build:types && npm run build:lib",
+    "start": "flow-copy-source --watch --verbose src lib & babel --watch --out-dir lib src"
```

Comment:
Would it be possible to also add the type checking to the npm start command? It would be nice to be warned if there are any type errors introduced.

Reply:
In practice there should be impossible to introduce type errors as that would fail In CI and there for won’t be merged.

Also given that initail run of type checking takes quite bit time I don’t think running that on start would be a good option.